### DISCUSSION TITLE
Task :react-native-bootsplash:compileDebugJavaWithJavac FAILED

### DISCUSSION THREAD

**Discussion by Nagakumar2402** [zoontek/react-native-bootsplash#556] [Q&A]
Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.
You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

**Comment by zoontek** [comment]
React Native doesn't support Gradle 9 yet (even 0.74 will not: https://github.com/facebook/react-native/blob/57cc8e7f6772fee1e01e9997c04ee66016495f83/packages/react-native/template/android/gradle/wrapper/gradle-wrapper.properties#L3)
